packages/@aws-cdk/aws-events-targets/lib/firehose-stream.ts

```diff
@@ -0,0 +1,47 @@
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as firehose from '@aws-cdk/aws-kinesisfirehose';
+import { singletonEventRole } from './util';
+
+/**
+ * Customize the Firehose Stream Event Target
+ */
+export interface FirehoseStreamProps {
+  /**
+   * The message to send to the stream.
+   *
+   * Must be a valid JSON text passed to the target stream.
+   *
+   * @default - the entire CloudWatch event
+   */
+  readonly message?: events.RuleTargetInput;
+}
+
+
+/**
+ * Customize the Firehose Stream Event Target
+ */
+export class FirehoseStream implements events.IRuleTarget {
+
+  constructor(private readonly stream: firehose.CfnDeliveryStream, private readonly props: FirehoseStreamProps = {}) {
+  }
+
+  /**
+   * Returns a RuleTarget that can be used to trigger this Firehose Stream as a
+   * result from a CloudWatch event.
+   */
+  public bind(_rule: events.IRule, _id?: string): events.RuleTargetConfig {
+    const policyStatements = [new iam.PolicyStatement({
+      actions: ['firehose:PutRecord', 'firehose:PutRecordBatch'],
```

Comment:
ah i'm guessing this is why you named the class `Firehose*`.
I still think it's probably good to align with our module name (KinesisFirehose)